README.md

```diff
@@ -58,3 +58,14 @@ manager.
 
 See [DEVELOPERS.md](https://github.com/BitGo/BitGoJS/blob/master/DEVELOPERS.md)
 
+
+# :muscle: Some common tips for developers onboarding to SDK: :muscle:
```

Comment:
Nice, this is good info, but I think this would be more appropriate in the DEVELOPERS.md file.